osu.Game/Overlays/FullscreenOverlay.cs

```diff
@@ -41,6 +39,23 @@ protected FullscreenOverlay()
             };
         }
 
+        private DependencyContainer dependencies;
+
+        protected override IReadOnlyDependencyContainer CreateChildDependencies(IReadOnlyDependencyContainer parent) =>
```

Comment:
just use `[Cached]` instead of this. much simpler.

Reply:
I thought about it but looks like I'm missing something because I can't cache like  that since colourScheme isn't static
![Аннотация 2020-01-26 123510](https://user-images.githubusercontent.com/22874522/73133345-52387780-4038-11ea-8be1-5bc6b0a2c9ea.png)

Reply:
Nevermind